clients/restclient/rest_client.go

```diff
@@ -79,6 +81,12 @@ func getRestURL(org, space string) string {
 	return "rest/" + org + "/" + space
 }
 
+func getRestURLEncoded(org, space string) string {
+	orgEncoded := strings.Replace(url.QueryEscape(org), "+", "%20", -1)
+	spaceEncoded := strings.Replace(url.QueryEscape(space), "+", "%20", -1)
+	return "rest/" + orgEncoded + "/" + spaceEncoded
```

Comment:
Could some builder be introduced and reused in the other methods?

Reply:
Only original function getRestURL is kept